voyager_archive/views_utils.py

```diff
@@ -1,14 +1,36 @@
 from django.shortcuts import get_object_or_404
+from pymarc import Record
 from .models import AuthRecord, BibRecord, MfhdRecord
 
 
 def get_auth_record(auth_id: int) -> AuthRecord:
```

Comment:
I've been using type hints in Python and want to continue throughout this project.  Python doesn't care but IDEs like VS Code use them well.  The return type on these should now be `list`, as that's what `Record.get_fields()` returns.